sonar-core/src/main/java/org/sonar/core/config/CorePropertyDefinitions.java

```diff
@@ -282,6 +285,14 @@ private CorePropertyDefinitions() {
         .category(CoreProperties.CATEGORY_EXCLUSIONS)
         .subCategory(CoreProperties.SUBCATEGORY_DUPLICATIONS_EXCLUSIONS)
         .multiValues(true)
+        .build(),
+
+      // ORGANIZATIONS
+      PropertyDefinition.builder(ORGANIZATIONS_ANYONE_CAN_CREATE)
+        .defaultValue(Boolean.toString(ORGANIZATIONS_ANYONE_CAN_CREATE_DEFAULT_VALUE))
```

Comment:
Ok, I didn't get you were talking about the constant for the default value and not the one of the name.
I'll remove the constant for the default value